examples/functions/trip/TR_08_response_dog.xml

```diff
@@ -0,0 +1,330 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OJP xmlns="http://www.vdv.de/ojp" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.1-dev" xsi:schemaLocation="http://www.vdv.de/ojp ../../../OJP.xsd">
+	<!-- as publicar is not allowed for a senior with dog (in the example) again the regular public transport is shown -->
+	<OJPResponse>
+		<siri:ServiceDelivery>
+			<siri:ResponseTimestamp>2023-09-13T14:02:49Z</siri:ResponseTimestamp>
+			<siri:ProducerRef>EFAController10.5.23.58-OJP-EFA01-P</siri:ProducerRef>
+			<siri:Status>true</siri:Status>
+			<siri:ErrorCondition>
+				<OJPError>
+					<TripProblemType>TRIP_RESTRICTEDRESULTSDROPPED</TripProblemType>
+				</OJPError>
```

Comment:
I think most people will not implement this. I think this can be used, when the service knows it dropped things because they are restricted.  In this case the Publicar was droopped, because: (a) no Senior (b) no IncludeRestrictedLines.

Reply:
OK.

Reply:
I improved the example some more.

Reply:
I think the PubliCar shouldn't be dropped, since ,,, "If multiple PassengerCategories are provided, then Services are shown when they are available for at least one PassengerCategory." 

And in that case the error condition wouldn't appear and should be shown in another example.

Reply:
now only dog, even when this doesn't make completly sense.